Gopkg.toml

```diff
@@ -118,5 +118,7 @@
   name = "github.com/ethereumproject/benchmark"
 
 [[constraint]]
-  branch = "master"
-  name = "github.com/fatih/color"
\ No newline at end of file
+  name = "github.com/fatih/color"
+  # TODO(tzdybal): monitor https://github.com/fatih/color/pull/87
+  source = "github.com/achilleas-k/color"
+  branch = "expose-colorised-stderr"
```

Comment:
Is contraining to `branch` ok? Or maybe tag or commit hash would be more solid?

Reply:
Remember about `Gopkg.lock` file - it stable until explicit call to `dep update`. And this is temporary solution anyways.

Reply:
Ah, just noticed

> solution: temporary switch to forked branch with proposed PR
with code that we require. After fatih/color#87
is merged, we can switch to original repository.

LGTM